### PR TITLE
MSDKUI-1672: Fix Speed Unit for UK

### PR DESCRIPTION
**Pull request by otaviohere**
In Swift, `Locale` has a property called `.usesMetricSystem` which is a Boolean. For UK, it returns true since UK uses a mix of Imperial and Metric System. 

In Objective-C, `NSLocal` has 3 possible values for metric system, US, UK, Metric. This commit uses the Objective-C bridge to check the `NSLocale`'s measurement system.

### CODE REVIEW

File: ExtensionLocale.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+extension Locale {
+
+    /// Returns a boolean indicating if the locale uses km/h for speed.
```
Comment (datasun) [inline ExtensionLocale.swift]: Maybe, instead of "km/h" say: a metric system?

File: ExtensionLocale.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
+//
```
Comment (piotrekm44) [inline ExtensionLocale.swift]: Please move from msdkui-ios/ExtensionLocale.swift to msdkui-ios/MSDKUI/Classes/ExtensionLocale.swift

Comment (otaviohere) [reply ExtensionLocale.swift]: Ouch ;-) Thanks. Done.

File: MSDKUI_Tests/ExtensionLocaleTests.swift [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+@testable import MSDKUI
+import XCTest
+
+final class ExtensionLocaleTests: XCTestCase {
+
+    /// Tests the `.usesKilometersPerHour` behavior for Germany.
+    func testUsesKilometersPerHourForGermany() {
+        let locale = Locale(identifier: "de_DE")
+        XCTAssertTrue(locale.usesKilometersPerHour, "It uses km/h for speed.")
```
Comment (piotrekm44) [inline MSDKUI_Tests/ExtensionLocaleTests.swift]: Please consider creating `Locale` object in place `XCTAssertTrue(Locale(identifier: "de_DE").usesKilometersPerHour, ...` instead an additional constant `let locale = ...`.

For example like in `GuidanceSpeedMonitorTests.testWhenCurrentSpeedIsKnown()`:
`XCTAssertEqual(mockDelegate.lastCurrentSpeed, Measurement(value: 100, unit: UnitSpeed.metersPerSecond), ...`

**Pull request merged**